### PR TITLE
Fix spacing in our informational banners

### PR DESCRIPTION
**Pull request by JasonLin0991**
## Summary

Adding more top and bottom spacing for the text in our informational banners: unlocked. This is to address comments in this ticket [MCR-4590](https://jiraent.cms.gov/browse/MCR-4590)

### CODE REVIEW

File: services/app-web/src/pages/SubmissionSummary/SubmissionSummary.tsx [hunk 1]
```diff
@@ -274,7 +274,7 @@ export const SubmissionSummary = (): React.ReactElement => {
             >
                 {renderStatusAlerts()}
 
-                {documentError && (
+                {!documentError && (
```
Comment (pearl-truss) [inline services/app-web/src/pages/SubmissionSummary/SubmissionSummary.tsx:277]: Is this intentional? I think we would want it to show when there's an error

Comment (JasonLin0991) [inline services/app-web/src/pages/SubmissionSummary/SubmissionSummary.tsx:277]:
```suggestion
                {documentError && (
```

Comment (JasonLin0991) [reply services/app-web/src/pages/SubmissionSummary/SubmissionSummary.tsx]: Oops that was for making the banner show. Good catch

**Review by pearl-truss — APPROVED** [review]
Thanks for the change, LGTM

**Pull request merged**